src/cc65/symtab.c

```diff
@@ -682,13 +682,21 @@ static void AddSymEntry (SymTable* T, SymEntry* S)
 
 
 
-SymEntry* AddEnumSym (const char* Name, const Type* Type, SymTable* Tab)
+SymEntry* AddEnumSym (const char* Name, unsigned Flags, const Type* Type, SymTable* Tab)
 /* Add an enum entry and return it */
 {
     SymTable* CurTagTab = TagTab;
+    SymEntry* Entry;
+
+    if ((Flags & SC_FICTITIOUS) == 0) {
+        /* Do we have an entry with this name already? */
+        Entry = FindSymInTable(CurTagTab, Name, HashStr(Name));
```

Comment:
Style nit: the space is missing in front of each left parenthesis.

Reply:
There are two lines of them. FIxed.